tests/integration-tests/tests/common/utils.py

```diff
@@ -401,3 +401,40 @@ def wait_process_completion(remote_command_executor, pid):
         raise Exception("The process is still running")
     else:
         return result.stdout.strip()
+
+
+def check_file_handler_leak(remote_command_executor, slurm_commands, cluster, region):
```

Comment:
I'm not sure this function name best describes what this is doing.  It seems like this just gets the number of open files.

Reply:
True, I renamed it to be `get_compute_ip_to_no_files`

Reply:
nit: I think the term `no` is less explicit than `num` to abbreviate the number of files.  `no` has more binary implications or boolean implications